src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,344 @@
+#What is the OpenHDS? 
+
+######It is the Open Health and Demographic System. It provides an intuitive and easy way to record populations and their changes over time. The OpenHDS logically breaks down the geography of an area of study and captures the major demographic events within it.
+
+##The OpenHDS view of the world:
+####The Hierarchy
+
+The first thing to understand is the way the OpenHDS represents the area of study for a project.
+For every geographical location on Earth there exists some way to represent that location hierarchically, much like is done when writing a letter to someone. In general, an address starts from the most specific description of the location and slowly become more general until the location you’re sending the letter to is well enough understood by the postal service. 
+
+The OpenHDS has a similar view of the world.
+
+The regions, districts, prefectures, cities, etc that comprise the area of study are each represented
+in the OpenHDS by an entity called **LocationHierarchy**. There is one *Root* LocationHierarchy that represents
+the largest geographical denomination for the area of study. Underneath the *Root* is a LocationHierarchy for every other geographical denomination, each of which points to their ‘parent’ which forms a tree-like structure that represents the entire area of study. The easiest way to understand this is with a picture using a real world example…
+
+#####The United States as modeled by LocationHierachy.    
+![Figure 1](figure-1.png)
+*figure 1*
+
+This is a simplified example. It’s obvious that there is not enough space on this page to list out all 50 states and all cities and streets for each state, but it perfectly demonstrates the aforementioned tree.
+
+####Everything Else
+
+Now that the study area is broken down logically into a hierarchy within the OpenHDS the demographic data has a geographical context. Every piece of information within the OpenHDS is tied in some way to the hierarchy, just as a population and its changes over time are tied to their geographical location. 
+
+The information in the OpenHDS comes in 3 main flavors. *Census* data: people, the places they live and the groups they belong to. *Relation* data: information about how people are related to one another, members of groups, or resident at a location. *Update* data: records of events that change the census data over time.
+
+#####A simplified overview of the OpenHDS entities and events.
+![Figure 2](figure-2.png)

```

Comment:
I agree with this breakdown, including the idea of Relation Entities.  I like the idea of just stacking up the entities as a summary, and leaving all the complex arrows and layout stuff for later.

What would you think of adding the rest of the update events?  Maybe each one would have to be half the size.  Then this figure would be an at-a-glance summary of the whole enchilada.  It's so close already.